modules/crio_trouble.adoc

```diff
@@ -0,0 +1,431 @@
+85418badbf6ae quay.io/coreos/configmap-reload...   4 hours ago Running config-reloader      0
+756f20138381c docker.io/openshift/prometheus-al... 4 hours ago Running alertmanager         0
+5e6d8ff4852ba quay.io/coreos/cluster-monitoring... 4 hours ago Running cluster-monitoring-  0
+1c96cfcfa10a7 docker.io/openshift/origin-haprox... 5 hours ago Running route                0
+8f90bb4cded60 docker.io/openshift/origin-node...   5 hours ago Running sdn                  0
+59e5fb8514262 docker.io/openshift/origin-node...   5 hours ago Running openvswitch          0
+73323a2c26abe docker.io/openshift/origin-node...   5 hours ago Running sync                 0
+```
+To see both running containers as well as containers that are stopped or
+exited, use `crictl ps -a`:
+```
+$ sudo crictl ps -a
+```
+
+If your CRI-O service is stopped or otherwise not working, you can list the
+containers that were run in CRI-O using the `runc` command. This example
+searches for the existence of a container with CRI-O running and
+not running. It then shows that you can investigate that container with
+`runc`, even when CRI-O is stopped:
+
+```
+$ crictl ps | grep d36a99a9a40ec
+d36a99a9a40ec       062cd20609d3895658e54e5f367b9d70f42db4f86ca14bae7309512c7e0777fd
+    11 hours ago        CONTAINER_RUNNING   sync                 2
+$ sudo systemctl stop crio
+$ sudo crictl ps | grep d36a99a9a40ec
+2018/10/25 11:22:16 grpc: addrConn.resetTransport failed to create client transport:
+   connection error: desc = "transport: dial unix /var/run/crio/crio.sock: connect:
+   no such file or directory"; Reconnecting to {/var/run/crio/crio.sock <nil>}
+   FATA[0000] listing containers failed: rpc error: code = Unavailable desc = grpc:
+   the connection is unavailable
+$ sudo runc list | grep d36a99a9a40ec
+d36a99a9a40ecc4c830f10ed2d5bb3ce1c6deadcb1a4879ff342e315051a71ed   19477       running
+  /run/containers/storage/overlay-containers/d36a99a9a40ecc4c830f10ed2d5bb3ce1c6deadcb1a4879ff342e315051a71ed/userdata
+  2018-10-25T04:44:29.47950187Z    root
+$ ls /run/containers/storage/overlay-containers/d36*/userdata/
+attach  config.json  ctl  pidfile  run
+$ less /run/containers/storage/overlay-containers/d36*/userdata/config.json
+{
+	"ociVersion": "1.0.0",
+	"process": {
+		"user": {
+			"uid": 0,
+			"gid": 0
+		},
+		"args": [
+			"/bin/bash",
+			"-c",
+			"#!/bin/bash\nset -euo pipefail\n\n# set by the node
+                        image\nunset KUBECONFIG\n\ntrap 'kill $(jobs -p);
+                        exit 0' TERM\n\n# track the current state of the ...
+$ sudo systemctl start crio
+```
+As you can see, even with the CRI-O service off, runc shows the existence of the
+container and its location in the file system, in case you want to look into it further.
+
+=== Investigating images, pods and containers
+To find out details about what is happening inside of images, pods or containers
```

Comment:
s/pods/pods,

Reply:
You obviously have an English and not a journalism background. :-)  I changed it as you suggested.